code/ui/blocks/src/components/Typeset.tsx

```diff
@@ -35,7 +35,7 @@ const Wrapper = styled.div(withReset, ({ theme }) => ({
 
 export interface TypesetProps {
   fontFamily?: string;
-  fontSizes: string[];
+  fontSizes: number[];
```

Comment:
The proper type would actually be `(string | number)[]` because that's the type of the underlying data model. It would be great if you could update the docs to reflect this as well.

```suggestion
  fontSizes: (string | number)[];
```

Reply:
@JReinhold Thanks! All set!